modules/shared/src/main/scala/org/tesselation/cli/env.scala

```diff
@@ -0,0 +1,77 @@
+package org.tesselation.cli
+
+import cats.syntax.contravariantSemigroupal._
+
+import org.tesselation.ext.decline.decline._
+
+import ciris.Secret
+import com.monovore.decline._
+import fs2.io.file.Path
+import io.estatico.newtype.macros.newtype
+import io.estatico.newtype.ops._
+
+object env {
+
+  sealed trait EnvOpt
+
+  @newtype
+  case class Password(value: Secret[String])
+
+  object Password {
+
+    val opts: Opts[Password] = Opts
+      .env[Password]("CL_PASSWORD", help = "Password")
+  }
+
+  @newtype
+  case class StorePass(value: Secret[String])
+
+  object StorePass {
+
+    val opts: Opts[StorePass] = Opts
+      .env[StorePass]("CL_STOREPASS", help = "Keystore password")
+  }
+
+  @newtype
+  case class KeyPass(value: Secret[String])
+
+  object KeyPass {
+
+    val opts: Opts[KeyPass] = Opts
+      .env[KeyPass]("CL_KEYPASS", help = "Key password")
+  }
+
+  @newtype
+  case class KeyAlias(value: Secret[String])
+
+  object KeyAlias {
+
+    val opts: Opts[KeyAlias] = Opts
+      .env[KeyAlias]("CL_KEYALIAS", help = "Alias of key in keystore")
+  }
+
+  @newtype
+  case class StorePath(value: Path)
+
+  object StorePath {
+
+    val opts: Opts[StorePath] = Opts
+      .env[Path]("CL_KEYSTORE", help = "Keystore path")
+      .map(_.coerce)
+  }
+
+  case class EnvConfig(
+    keystore: StorePath,
+    storepass: StorePass,
+    keypass: KeyPass,
```

Comment:
can we unify it into `password` or we need it to support legacy keys?

Reply:
We need to support legacy keys in `wallet` but we force `password` in `keytool` already